docs/frontend/nx-monorepo.md

```diff
@@ -70,6 +70,29 @@ A set of helm templates have been created in `{repo}/helm/cas-registration/templ
 
 ## Nx Generators 🦾
 
+### Creating new projects 🏗
+
+For further information, see the [Nx docs about @nx/next generators](https://nx.dev/nx-api/next/generators/application).
+
+#### Eslint
+
+When creating a new project with Nx, there is some setup that needs to be done to ensure that eslint works correctly with typescript. Nx doesn't set this up by default since there is a performance hit though we are using typescript in all of our projects. More information about this can be found in the [Nx docs](https://nx.dev/latest/react/guides/eslint).
+
+This is done by adding the following to the `apps/{project}/.eslintrc.json` or `libs/shared/{project}/.eslintrc.json` file:
```

Comment:
Suggest adding how to know which of these two files to edit. Also, if the new project doesn't need any custom rules, can it use the bciers config instead of having its own?

Reply:
Well these are generated by nx with the project setup. I think there's an argument to be made to remove them and I thought about it, though instead I moved all the rules to the main `eslintrc` and left these in.  
  
I think there are good reasons to keep these in and allow customization of the rules per project as there may be rules that work better for some projects than others. Though largely I think it's good to keep them in incase we have a team agreement to change a global rule, there may be older projects that it's not worth updating or projects where that rule doesn't make sense. This way we can turn it off.